cmd/swagger/setup.go

```diff
@@ -73,16 +76,44 @@ func SetupAPI(entClient *ent.Client, lg *zap.Logger, conf *config.AppConfig) (*r
 	server := restapi.NewServer(api)
 	listeners := []string{"http"}
 
+	server.ConfigureAPI()
 	server.EnabledListeners = listeners
 	server.Host = conf.Server.Host
 	server.Port = conf.Server.Port
+
+	accessManager, err := AccessManager(api, conf.AccessBindings)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to create access manager: %w", err)
+	}
 	server.SetHandler(
 		cors.AllowAll().Handler(
-			middlewares.Tx(entClient)(api.Serve(nil)),
+			middlewares.Tx(entClient)(api.Serve(accessManager.Middleware())),
 		),
 	)
 
 	return server,
 		overdue.NewOverdueCheckup(orderStatusRepo, orderFilterRepo, equipmentStatusRepo, lg),
 		nil
 }
+
+func AccessManager(api *operations.BeAPI, bindings []config.RoleEndpointBinding) (*middlewares.AccessManager, error) {
+	roles := []string{
```

Comment:
Should we make the role list and full access role list configurable?

Reply:
current state is fine from my side.